test-end-to-end/android-maestro/common/add_module_block.yaml

```diff
@@ -6,18 +6,18 @@ appId: com.toughra.ustadmobile
   # 2) MODULETITLE : provide the title for module block of String type
 
 # add module
-- repeat:
-    while:
-      notVisible: "Add block"
-    commands:
-      - swipe:
-          start: 100,400
-          end: 100,100
+- scrollUntilVisible:
+    element:
+      text: "Add block"
+    direction: DOWN
 - tapOn: "Add block"
 - tapOn: "Module"
-- tapOn: "Title"
+- tapOn: "Title*"
 - inputText: ${MODULETITLE}
-- tapOn: "Don't show before (Optional)"
-- tapOn: "OK"
+- tapOn: "Description"
+- tapOn:
+    id: "com.toughra.ustadmobile:id/editor"
+- inputText: "test block"
+- tapOn: "Done"
 - takeScreenshot: "${TESTRESULTSDIR}/screenshots/added_module_${MODULETITLE}"
-- tapOn: "SAVE"
+- tapOn: "Done"
```

Comment:
This looks like tapping "done" twice in a row. We should add an assertion to make sure that the screen has changed (e.g. check the title has changed)